src/bp/core/config/botpress.config.ts

```diff
@@ -452,6 +452,12 @@ export interface AuthStrategy {
    * @default false
    */
   allowSelfSignup: boolean
+
+  /**
+   * When enabled, authentication type will not be shown on a login page and only alternative one will be present.
+   * @default false
+   */
+  hide: boolean
```

Comment:
I'm a bit picky, but don't you think this property should be named "hidden" ?

Reply:
Yeah, now I saw the general pattern for other config variables and I think you are right. Will change that, thanks!